cmd/restart.go

```diff
@@ -0,0 +1,62 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/docker/libcompose/docker"
+	"github.com/docker/libcompose/docker/ctx"
+	"github.com/docker/libcompose/project"
+	"github.com/spf13/cobra"
+)
+
+// restartCmd represents the restart command
+var restartCmd = &cobra.Command{
+	Use:   "restart",
+	Short: "Restart a Shipment",
+	Long:  `Adds a dummy variable to the shipment and triggers it.`,
+	Run:   restart,
+}
+
+func init() {
+	RootCmd.AddCommand(restartCmd)
+}
+
+// restart your shipment
+func restart(cmd *cobra.Command, args []string) {
+
+	//make sure user is authenticated
+	username, token, err := Login()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	//read the harbor compose file
+	harborCompose := DeserializeHarborCompose(HarborComposeFile)
```

Comment:
You can replace the 2 calls to unmarshal the compose files with a call to `unmarshalCompose` which returns both.